### PR TITLE
Added test for yanking

### PR DESCRIPTION
**Pull request by julianuu**

### CODE REVIEW

File: tests/end2end/data/hints/simple2.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+
+<!-- target: hello.txt -->
```
Comment (The-Compiler) [inline tests/end2end/data/hints/simple2.html]: No need for that comment, for the same reason as [before](https://github.com/The-Compiler/qutebrowser/pull/1842#discussion_r74444550) :wink:

File: tests/end2end/features/hints.feature [hunk 2]
```diff
@@ -81,6 +81,15 @@ Feature: Using hints
...
 
+    Scenario: Yanking
+        When selection is supported
+        And I run :debug-set-fake-clipboard
+        And I open data/hints/simple2.html
+        And I hint with args "links yank" and follow a
+        And I hint with args "links yank-primary" and follow s
+        Then the clipboard should contain "http://localhost:(port)/data/hello.txt"
+        And the primary selection should contain "http://localhost:(port)/data/hello2.txt"
+
```
Comment (The-Compiler) [inline tests/end2end/features/hints.feature]: Why not separate this into two tests (clipboard/selection)? Then the clipboard will be tested on Windows/OS X as well.

**Comment by julianuu** [comment]
Oops for some reason I made a second pull request instead of updating this one. I am new to git so I don't exactly know what I did. How to fix this?

File: tests/end2end/features/hints.feature (hunk 2 above)
Comment (julianuu) [inline tests/end2end/features/hints.feature]: I thought it would save... idk... space? But yeah, maybe thats not really important

**Comment by The-Compiler** [comment]
Not sure what happened exactly - it seems you switched to working from a branch instead of commiting in `master` (which is a good thing, I try to keep `master` in sync with the `master` of whatever project I'm contributing to, and only commit in branches).

In the other PR, the `.html` file is missing, so I suggest you do something like:

```
$ git checkout test_yanking
$ git checkout master -- tests/end2end/data/hints/simple2.html  # get missing file from the master branch
[edit the file to remove the `<!-- target ... -->` comment again]
$ git commit -am "tests: Add data/hints/simple2.html"
$ git push
```

At that point, we can close this PR and continue in the new one.

**Comment by mlochbaum** [comment]
To update a PR, push to the branch that you have requested be pulled. This branch is shown at the top ("julianuu  wants to merge 2 commits into The-Compiler:master from julianuu:master").

You created a new branch `test-yanking` for the second PR, which is the correct thing to do. Provided this is a tracking branch, you can update that PR simply by committing while in the `test-yanking` branch and running `git push`. If it isn't, then `git push` will give an error. I recommend reading https://git-scm.com/book/en/v2/Git-Branching-Remote-Branches to learn how tracking branches work.

Once you have `test-yanking` sorted out, you should probably reset your master branch to match the one here. This destroys the old work on `master` (recoverably, but it's a hassle to undo), so close this PR and make sure everything is moved to `test-yanking` first. Then:
```bash
$ git fetch upstream
$ git reset --hard upstream/master
$ git push --force origin master
```
After that, your `master` branch will be the same as the upstream branch, and you can create new PRs by making new branches from `master`, pushing them to your fork of qutebrowser, and pull requesting through Github.

**Comment by The-Compiler** [comment]
Oh, sorry - just now I realized that #1892 actually doesn't need `simple2.html` anymore, so disregard what I said above :wink:

As for @mlochbaum's explanation (which is great!): That requires that you previously added my repo as `upstream` remote, with something like `git remote add upstream https://github.com/The-Compiler/qutebrowser.git`.

I'll close this PR now, as this is obsoleted by #1892.

**Comment by julianuu** [comment]
Yeah, thanks, your explanations did help a lot! And sorry for the late reply

**Pull request closed without merging**